src/ImageProcessorCore/Formats/Tiff/TiffExifExtractor.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace ImageProcessorCore.Formats
+{
+    /// <summary>
+    ///
+    /// </summary>
+    internal class TiffExifExtractor : ITiffVisitor
+    {
+        private readonly Dictionary<ushort, TiffTag> _exifTags; 
+
+        public List<ImageProperty> Properties { get; private set; }
+
+        public TiffExifExtractor()
+        {
+            Properties = new List<ImageProperty>();
+            
+            _exifTags = TiffTagRegistry.Instance.Tags
+                .Where( i => i.TagGroup == "Exif")
+                .ToDictionary(i => i.TagId);
+
+        }
+
+        public void Visit(TiffProperty property)
+        {
+            if (_exifTags.ContainsKey(property.Tag.TagId))
+            {
+                // until the image property can hold more than just a string value....
+                ImageProperty imageProperty = new ImageProperty(property.Tag.Name, property.Value.ToString());
```

Comment:
That old chestnut yeah... I've dealt with the orientation property before normalizing rotation from iPhone images.

I imagine we would have to do a lot of conversion to usable properties. How would you consider going about that with so much variance? 

I think I should probably be using the concrete Image class in the decoders. There's really no reason for us to decode to anything else.